### PR TITLE
 Updated specs

### PR DESCRIPTION
**Pull request by jc-berger**
For issue [#158](https://github.com/devfile/api/issues/158)

### CODE REVIEW

File: docs/modules/ROOT/partials/proc_writing-a-devfile-for-your-project.adoc [hunk 1]
```diff
@@ -43,7 +43,8 @@ projects:
       location: 'https://github.com/spring-projects/spring-petclinic.git'
       branch: master
 components:
```
Comment (elsony) [inline docs/modules/ROOT/partials/proc_writing-a-devfile-for-your-project.adoc:45]: It should be changed to:
```
components:
  - name: javaPlugin
    plugin:
      id: redhat/java/latest
```

**Review by elsony — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/partials/proc_writing-a-devfile-for-your-project.adoc (hunk 1 above)
Comment (jc-berger) [reply docs/modules/ROOT/partials/proc_writing-a-devfile-for-your-project.adoc:45]: addressed, so resolved conversation

**Comment by jc-berger** [comment]
@elsony, thanks, luckily that was a quick fix! Revised commit is up, thanks.

**Review by elsony — APPROVED** [review]
/lgtm

**Pull request merged**